src/reading.rs

```diff
@@ -1138,7 +1143,19 @@ pub mod async_api {
 		}
 	}
 
-	impl<T :AsyncRead> Stream for PacketReader<T> {
+	impl<T :FuturesAsyncRead> PacketReader<Compat<T>> {
+		/// Wraps the specified futures_io `AsyncRead` into an Ogg packet
+		/// reader. This crate uses Tokio internally, so a wrapper that
```

Comment:
Can you put an empty line between the `reader.` and `This`? This is so that it gets displayed nicely rustdoc.

https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components

Reply:
Done, thank you!